planet/cli/orders.py

```diff
@@ -269,20 +296,39 @@ def read_file_json(ctx, param, value):
               help='Toolchain json file.',
               type=click.File('rb'),
               callback=read_file_json)
+@click.option('--email',
+              default=False,
+              is_flag=True,
+              help='Send email notification when Order is complete')
+@click.option(
+    '--like',
+    type=click.STRING,
+    help='File or stdin providing the order description to use as a template.',
+    required=False)
+@click.option('--cloudconfig',
+              help='Cloud delivery config json file.',
```

Comment:
@kevinlacaille `cloudconfig` is a little opaque to me and users might need a little more explanation. I don't see the term used in our API docs.